pkg/keepalive/options.go

```diff
@@ -44,6 +44,14 @@ type Options struct {
 	// MaxServerConnectionAgeGrace is an additive period after MaxServerConnectionAge
 	// after which the connection will be forcibly closed by the server.
 	MaxServerConnectionAgeGrace time.Duration // default value 10s
+	// WriteBufferSize determines how much data can be batched before doing a write on the wire.
+	// The corresponding memory allocation for this buffer will be twice the size to keep syscalls low.
+	// The default value for this buffer is 32KB.
+	WriteBufferSize int
```

Comment:
These two option does not belong to keepalive. How about changing the pkg to grpc? And also move all grpc related options in.

Reply:
I am not quite sure about the usage of some other parameters in grpc options. Can I just change the package name and add a todo tag?